Resources/doc/reference/configuration/headers.rst

```diff
@@ -245,6 +261,9 @@ This enables a simplistic ETag calculated as md5 hash of the response body:
     page. If the page depends on permissions, make sure to make the ETag differ
     based on those permissions (e.g. by appending the :doc:`user context hash </features/user-context>`).
 
+    You may want to set your ETags weak when you are using gzip compression on server.
```

Comment:
is the server the webserver, or the reverse proxy? i would prefer to be more explicit here, maybe also say why that is, cite the RFC that talks about this or some other further info.

and "may" or "should"?